win32ss/user/user32/misc/imm.c

```diff
@@ -108,6 +108,18 @@ BOOL WINAPI User32InitializeImmEntryTable(DWORD magic)
     if (!InitializeImmEntryTable())
         return FALSE;
 
+    if (ghImm32 == NULL)
```

Comment:
This looks like duplication of what `InitializeImmEntryTable` already is doing?
Is there a reason to retry it again here?

Reply:
Here, `GetModuleHandleW` doesn't update the global variable `ghImm32`. So, we can detect whether `user32` did explicitly load `imm32` or not by checking `ghImm32`.

Reply:
Note: ghImm32 is to store the handle corresponding to what is returned from the LoadLibrary("imm32.dll") so that uuser32 knows it can then free it later.